trustfall_stubgen/src/edges_creator.rs

```diff
@@ -4,6 +4,8 @@ use maplit::btreemap;
 use quote::quote;
 use trustfall::{Schema, SchemaAdapter, TryIntoStruct};
 
+use crate::util::escaped_rust_name;
```

Comment:
can you get this from `super::util::` below?

Reply:
[19db2ef](https://github.com/obi1kenobi/trustfall/pull/406/commits/19db2ef0228d3aecf9d8d78dcdb5d4220a1a6f3f)